### DISCUSSION TITLE
Show right border for navigation bar

### DISCUSSION THREAD

**Discussion by SoumayaMauthoorMOJ** [squidfunk/mkdocs-material#4565] [Q&A]
Is it possible to show a border on the right of the navigation bar? or on the left of the main contents?

**Comment by squidfunk** [comment]
Yes, [use additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css).

**Comment by SoumayaMauthoorMOJ** [comment]
Sorry I should have asked - how do I show a border  on the right of the navigation bar? or on the left of the main contents?
